dbschema/src/main/java/org/folio/rest/persist/ddlgen/ForeignKeys.java

```diff
@@ -129,7 +129,8 @@ public String toString() {
     return "ForeignKeys [tableAlias=" + tableAlias + ", targetTable=" + targetTable
         + ", targetTableAlias=" + targetTableAlias
         + ", fieldName=" + fieldName
-        + (targetPath == null ? "" : "targetPath=" + targetPath)
+        + (targetPath == null ? "" : ", targetPath=" + targetPath)
+        + ", tOps=" + gettOps()
```

Comment:
What the previous code at error ? Missing ","?

Reply:
toString() is used for debugging only. The comma doesn't have any influence on production code.

Reply:
ok